okhttp-client/src/main/scala/org/http4s/client/okhttp/OkHttpBuilder.scala

```diff
@@ -56,27 +56,22 @@ import scala.util.control.NonFatal
   * @param blockingExecutionContext $BLOCKINGEC
   */
 sealed abstract class OkHttpBuilder[F[_]] private (
-    val okHttpClient: OkHttpClient,
-    val blocker: Blocker
-)(implicit protected val F: ConcurrentEffect[F], cs: ContextShift[F])
+    val okHttpClient: OkHttpClient
+)(implicit protected val F: Async[F])
     extends BackendBuilder[F, Client[F]] {
   private[this] val logger = getLogger
 
+  private def invokeCallback[F[_]](logger: Logger)(f: => Unit)(implicit
+      F: Sync[F]): Unit =
+    F.blocking(f).unsafeRunSync
```

Comment:
Is there a Dispatcher floating around that I don't see? Also does this need to be blocking?

Reply:
Added the dispatcher and made a few changes.